apps/backoffice/src/lib/be/institutions/business.ts

```diff
@@ -23,6 +33,15 @@ export const retrieveInstitution = async (
 ): Promise<BackofficeInstitution> =>
   toBackofficeInstitution(await getInstitutionById(institutionId));
 
+export const retrieveUserGroups = async (
```

Comment:
This method seems to retrieve all groups set for an institution, but it is named "retrieveUserGroups", what is the intent here?

Reply:
[4ce0326](https://github.com/pagopa/io-services-cms/pull/1007/commits/4ce0326ee02d74d1979064291f7fad04d82ab937)